src/crl.c

```diff
@@ -90,10 +90,16 @@ static int InitCRL_Entry(CRL_Entry* crle, DecodedCRL* dcrl, const byte* buff,
     XMEMCPY(crle->nextDate, dcrl->nextDate, MAX_DATE_SIZE);
     crle->lastDateFormat = dcrl->lastDateFormat;
     crle->nextDateFormat = dcrl->nextDateFormat;
+    crle->version = dcrl->version;
+    crle->issuer = wolfSSL_X509_NAME_new();
```

Comment:
Should you handle any `crle->issuer == NULL` case?

Reply:
Yes. Added.